electron/src/renderer/App.jsx

```diff
@@ -0,0 +1,104 @@
+import { useState, useEffect, useContext } from 'react';
+import {
+  MemoryRouter as Router,
+  Switch,
+  Route,
+  Link,
+  Redirect,
+  useHistory,
+  useLocation,
+} from 'react-router-dom';
+import './App.css';
+import { realmApp, Authentication } from './realm';
```

Comment:
remember to replace realmApp with the inbuilt getApp

Reply:
after reviewing the [method docs for `getApp()`](https://docs.mongodb.com/realm-sdks/js/latest/Realm.App.html#getApp) i'm not convinced that it would be better to use `getApp` instead of the current implementation. this is b.c the `getApp()` documentation requires passing in the app id as an argument every time it's called. 

so to use the `getApp()` approach, we'd need to either: 
1. define APP_ID as an environmental variable or variable to be imported, and pass it to every invocation of `getApp`
2. use the string of the App ID in every invocation, which would not be a very DRY solution

lmk what you think here

Reply:
Fair point on it not being as DRY, but we should define the App ID in a config file (not env). We can take the standard set out in the web & rn templates, of using realm.json: https://github.com/mongodb-university/realm-template-apps/blob/main/react-native/realm.json